frontend/src/MuiForms/EntitySelectorBeta.tsx

```diff
@@ -0,0 +1,82 @@
+import { Chip } from '@mui/material'
+import Autocomplete from '@mui/material/Autocomplete'
+import TextField from '@mui/material/TextField'
+import * as React from 'react'
+
+import { useGetCurrentUser, useListUsers } from '../../actions/user'
+import Loading from '../common/Loading'
+import MessageAlert from '../MessageAlert'
+
+interface EntitySelectorProps {
+  label?: string
+  required?: boolean
+  value: string[]
+  onChange: (newValue: string[]) => void
+}
+
+export default function EntitySelectorBeta(props: EntitySelectorProps) {
+  const { users, isUsersLoading: isLoading } = useListUsers()
+  const { currentUser, isCurrentUserLoading, isCurrentUserError } = useGetCurrentUser()
+  const [open, setOpen] = React.useState(false)
+
+  const entities = React.useMemo(() => {
+    let tempEntities: string[] = []
+
+    if (users) tempEntities = tempEntities.concat(users.map((user) => user.id))
+    return tempEntities
+  }, [users])
+
+  const { onChange, value: currentValue, required, label } = props
+
+  const _onChange = (_event: React.SyntheticEvent<Element, Event>, newValues: string[]) => {
```

Comment:
As we use `_` to denote unused params could we rename `_onChange` to `handleChange`?